sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformConfiguration.java

```diff
@@ -48,8 +48,14 @@ public void validate() {
     assert startOffset == null || VALID_START_OFFSET_VALUES.contains(startOffset)
         : "Valid Kafka Start offset values are " + VALID_START_OFFSET_VALUES;
     final String dataFormat = this.getFormat();
-    assert dataFormat == null || VALID_DATA_FORMATS.contains(dataFormat)
-        : "Valid data formats are " + VALID_DATA_FORMATS;
+    assert dataFormat == null || isValidDataFormat(dataFormat)
+        : "Valid data formats are " + VALID_FORMATS_STR;
```

Comment:
TBD on returning STR vs the SET.  I see pros/cons, so this is mostly to call out the thought process and choice/difference than a problem.

Reply:
On rereading, I guess I don't see the value of this change other than adding RAW?